pkg/scaffold/scaffold.go

```diff
@@ -154,7 +121,7 @@ func (s *Scaffold) defaultOptions(options *input.Options) error {
 
 	// Use the default Project path if unset
 	if options.ProjectPath == "" {
-		options.ProjectPath = "PROJECT"
+		options.ProjectPath = config.DefaultPath
 	}
```

Comment:
Could not we simplify the above in the new design?

Reply:
Several:

https://github.com/kubernetes-sigs/kubebuilder/blob/d98c4cb16ef88117500da3232f2007382d435c69/pkg/scaffold/v1/crd/suite_test.go#L99

https://github.com/kubernetes-sigs/kubebuilder/blob/d98c4cb16ef88117500da3232f2007382d435c69/pkg/scaffold/v1/manager/manager_test.go#L69

https://github.com/kubernetes-sigs/kubebuilder/blob/d98c4cb16ef88117500da3232f2007382d435c69/pkg/scaffold/v1/manager/manager_test.go#L82

https://github.com/kubernetes-sigs/kubebuilder/blob/d98c4cb16ef88117500da3232f2007382d435c69/pkg/scaffold/v1/webhook/webhook_test.go#L184

And `scaffoldtest.Options()` is:

https://github.com/kubernetes-sigs/kubebuilder/blob/d98c4cb16ef88117500da3232f2007382d435c69/pkg/scaffold/scaffoldtest/scaffoldtest.go#L44-L64

Reply:
Closing this one.